bait-files/src/main/java/com/google/sps/servlets/UserHistoryServlet.java

```diff
@@ -0,0 +1,73 @@
+package com.google.sps.servlets;
+
+import static java.util.stream.Collectors.toList;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.servlets.Enums.Status;
+import com.googlecode.objectify.ObjectifyService;
+import com.googlecode.objectify.cmd.Query;
+import java.io.IOException;
+import java.lang.String;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java. util.Comparator;
+import java. util.Collections;
```

Comment:
Remove these extra spaces between "java." and "util".

Reply:
Removed spaces